code/modules/mob/living/carbon/monkey/update_icons.dm

```diff
@@ -18,6 +18,19 @@
 
 /mob/living/carbon/monkey/regenerate_icons()
 	..()
+	var/icon/opacity_icon = new(icon, icon_state)
+	if(body_alphas.len)
+		opacity_icon.ChangeOpacity(ARBITRARILY_LARGE_NUMBER)
+		var/lowest_alpha = 255
```

Comment:
Don't particularly like how often this loop is copypasted. Make it a proc like `get_lowest_alpha()`.